### PR TITLE
Add options to repo manager's add-referenceset

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #1150

### CODE REVIEW

File: ga4gh/cli.py [hunk 1]
```diff
@@ -1953,6 +1990,11 @@ def getParser(cls):
             "file must be bgzipped and indexed.")
         cls.addNameOption(addReferenceSetParser, objectType)
         cls.addDescriptionOption(addReferenceSetParser)
+        cls.addNcbiTaxonIdOption(addReferenceSetParser)
```
Comment (jeromekelleher) [inline ga4gh/cli.py]: These aren't shared by any other parsers, so might as well just put them in directly without the class methods. Saves a few lines of code, and it's easier to read to see the overall layout of the parser.

**Comment by jeromekelleher** [comment]
Great, +1 modulo suggestion above.

**Comment by david4096** [comment]
Good stuff +1

**Comment by dcolligan** [comment]
@jeromekelleher updated

**Comment by david4096** [comment]
@dcolligan can you rebase and merge? Thanks!

**Pull request merged**